dev/Scroller/ScrollerPrivate.cpp

```diff
@@ -9,12 +9,12 @@
 
 bool Scroller::IsHorizontallyScrollable()
 {
-    return !m_isChildAvailableWidthConstrained;
+    return m_contentOrientation != winrt::ContentOrientation::Vertical;
```

Comment:
m_contentOrientation == winrt::ContentOrientation::Horiz?
otherwise maybe you need to m_contentOrientation != winrt::ContentOrientation::Vertical && m_contentOrientation != winrt::ContentOrientation::None;

Reply:
The old and new code are equivalent, because m_isChildAvailableWidthConstrained <==> m_contentOrientation == winrt::ContentOrientation::Vertical.